src/main/java/com/target/studentinfo/dto/validator/StudentValidator.java

```diff
@@ -1,26 +1,26 @@
 package com.target.studentinfo.dto.validator;
 
-import com.target.studentinfo.dto.request.StudentRequest;
 import com.target.studentinfo.exception.BadRequestException;
 import com.target.studentinfo.exception.ErrorCode;
-import org.springframework.http.HttpStatus;
+import com.target.studentinfo.model.Student;
 
 public class StudentValidator {
-    public static void validateStudentRequest(StudentRequest studentRequest) {
-        if (studentRequest == null) {
-            throw new BadRequestException(ErrorCode.INVALID_REQUEST_BODY, HttpStatus.BAD_REQUEST);
+    public static void validateStudentRequest(Student student) {
+        if (student == null) {
+            throw new BadRequestException(ErrorCode.INVALID_REQUEST_BODY, "request body cannot be null");
```

Comment:
```
public final class ErrorMessage {

    private ErrorMessage() {
    }

    public static final String INVALID_FIRST_NAME = "first name cannot be null";
}
```
Define a constant file like above for error message strings and refer strings from there. Keep the error message names same as error codes, it should be fine.